services/frontend-service/src/ui/ReleaseCard/ReleaseCard.tsx

```diff
@@ -0,0 +1,67 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2021 freiheit.com*/
+import classNames from 'classnames';
+import { Button } from '../components/button';
+import { useEffect, useRef } from 'react';
+import { MDCRipple } from '@material/ripple';
+import { Chip } from '../components/chip';
+
+export type ReleaseCardProps = {
+    className?: string;
+    title: string;
+    hash: string;
+    createdAt: Date;
+    author: string;
+    environments: string[];
+};
+
+export const ReleaseCard: React.FC<ReleaseCardProps> = (props) => {
+    const MDComponent = useRef<MDCRipple>();
+    const control = useRef<HTMLDivElement>(null);
+    const { className, title, hash, createdAt, author, environments } = props;
+
+    useEffect(() => {
+        if (control.current) {
+            MDComponent.current = new MDCRipple(control.current);
+        }
+        return () => MDComponent.current?.destroy();
+    }, []);
+
+    return (
+        <div className={classNames('mdc-card release-card', className)}>
+            <div className="release-card__header">
+                <div className="release__title mdc-typography--headline6">{title}</div>
+                <Button className="release__hash" label={hash} />
```

Comment:
The hash can be empty. We should probably hide this button then.

Reply:
I'm curious when this could happen? 
for the undeploy versions?